example/src/main/config/example-tenants.json

```diff
@@ -0,0 +1,29 @@
+[
+  {
+    "tenant": "DEFAULT_TENANT",
+    "data": {
+      "enabled": "true"
+    }
+  },
+  {
+    "tenant": "HTTP_TENANT",
+    "data": {
+      "enabled": "true"
+    },
+    "adapters": {
+      "http": {
```

Comment:
the type name of the HTTP adapter is *hono-http* (see https://www.eclipse.org/hono/user-guide/http-adapter/#downstream-meta-data)

Reply:
:+1: